conf/cassandra.yaml

```diff
@@ -960,6 +967,7 @@ server_encryption_options:
     # cipher_suites: [TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA,TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA]
     # require_client_auth: false
     # require_endpoint_verification: false
+    # outgoing_encrypted_port_source: yaml
```

Comment:
This field is not referenced anywhere in the code (not in `ServerEncrpytionOptions`). Is this just incomplete?

Reply:
It's out of date. I implemented this functionality, but then you implemented and merged competing functionality. I"ll remove it.